source/speech.py

```diff
@@ -1545,32 +1571,6 @@ def getFormatFieldSpeech(attrs,attrsCache=None,formatConfig=None,reason=None,uni
 					textList.append(u"%s %s"%(label,newVal))
 				else:
 					textList.append(noVal)
-		verticalAlign=attrs.get("vertical-align")
-		oldverticalAlign=attrsCache.get("vertical-align") if attrsCache is not None else None
-		if (verticalAlign or oldverticalAlign is not None) and verticalAlign!=oldverticalAlign:
-			verticalAlign=verticalAlign.lower() if verticalAlign else verticalAlign
-			if verticalAlign=="top":
-				# Translators: Reported when text is vertically top-aligned.

```

Comment:
@josephsl This code block has been moved. Could you comment on what the consequence of this would be for translators?

This actually resolves a bug, where the vertical alignment reporting is controlled via the paragraph indentation option in the document formatting settings.

Reply:
@josephsl: Could you address @feerrenrut's comment?

@feerrenrut: I belief you could check this quite easily by creating a new pot file from this branch and import that pot file into an up to date translation. If nothing starts being fuzzy, it's alright. :)

Reply:
I spoke with Mick about this, we have decided to leave this and the other similar PR until after the release of 2017.4